zbuild.h

```diff
@@ -190,7 +190,7 @@
 #  define UNLIKELY(x)           x
 #endif /* (un)likely */
 
-#if defined(__clang__) || defined(__GNUC__)
+#if defined(__clang__) || defined(__GNUC__) || defined(HAVE_ATTRIBUTE_ALIGNED)
```

Comment:
Does using `Z_HAS_ATTRIBUTE(aligned)` instead work for your use case?

Reply:
I think that would work.

Reply:
I think that would work.

Reply:
> Does using `Z_HAS_ATTRIBUTE(aligned)` instead work for your use case?

No. If a non-GCC (or Clang) compiler supports a subset of GCCisms such as `__attribute__(aligned(x))` for compatibility/portability reasons, it doesn't necessarily support `__has_attribute`.

The compiler I'm using supports a small subset of `__attribute__` values (including `aligned`) but does not support `__has_attribute`.